packages/testing/config/jest/api/jest-preset.js

```diff
@@ -0,0 +1,84 @@
+const path = require('path')
+
+const {
+  getPaths,
+  getApiSideDefaultBabelConfig,
+  getApiSideBabelPresets,
+  getApiSideBabelPlugins,
+} = require('@redwoodjs/internal')
+
+const rwjsPaths = getPaths()
+const NODE_MODULES_PATH = path.join(rwjsPaths.base, 'node_modules')
+
+// @TODO
```

Comment:
Any suggestions for a better way to do this?

Reply:
I like how this decouples the prisma reset from the CLI; I'll think about this more.